tokio/CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 0.2.21 (July 20, 2020)
+
+### Fixes
+
+- docs: misc improvements (#2572, #2658, #2663, #2656, #2647, #2630, #2487, #2621,
+  #2624, #2600, #2623, #2622, #2577, #2569, #2589, #2575, #2540, #2564, #2567,
+  #2520, #2521, #2572, #2493)
+- rt: allow calls to `block_on` inside calls to `block_in_place` that are
+  themselves inside `block_on` (#2645)
+- net: fix non-portable behavior when dropping `TcpStream` `OwnedWriteHalf` (#2597)
+- io: improve stack usage by allocating large buffers on directly on the heap
+  (#2634)
+- io: fix unsound pin projection in `AsyncReadExt::read_buf` and
+  `AsyncWriteExt::write_buf` (#2612)
+- io: fix unnecessary zeroing for `AsyncRead` implementors (#2525)
+- io: Fix `BufReader` not correctly forwarding `poll_write_buf` (#2654)
+
+### Changes
+- coop: returning `Poll::Pending` no longer decrements the task budget (#2549)
+
+### Added
+
+- io: little-endian variants of `AsyncReadExt` and `AsyncWriteExt` methods
+  (#1915)
+- io: fix panic in `AsyncReadExt::read_line` (#2541)
```

Comment:
This seems like it should go in the fixes section.

Reply:
whoops